tests/x11/libreoffice/libreoffice_recent_documents.pm

```diff
@@ -45,15 +45,17 @@ sub run {
     # Check Recent Documents
     wait_still_screen;
     x11_start_program('oowriter');
-    # Because of bsc#1074057 alt-f is not working in libreoffice under wayland
-    # use another way to replace alt-f in SLED15
-    if (is_sle '15+') {
+
+    wait_still_screen;
+    send_key "alt-f";
+    wait_still_screen;
```

Comment:
This will waste 7 seconds twice at least and should be avoided

Reply:
Both `wait_still_screen` are needed: first is to make sure libreoffice has finished initialization, second is to avoid false positive -- under wayland, the menu could show up and then disappear, so we have to `wait_still_screen` first.

Reply:
Of course you need proper synchronisation. But please don't waste 14 seconds. I suggest to simply use proper `assert_screen` calls.

Reply:
There is a `assert_screen` in `x11_start_program`, but the needle `ooffice-1-gnome-80p_match-20190208.png` has a 80% match threshold causes this initializing libreoffice to match the initialized libreoffice.  That needle was committed by you, but I think it was a workaround for staging and that needle should be deleted, right?

Reply:
I don't know why I commited the needle with 80% match at this time however in general I am one of the few to create needles with a match level lower than default to reduce the needle maintenance effort when the strictness of a higher match level is not required. If you want to prevent false-matches feel free to delete the needles while ensuring there are other needles that provide a match in the situations where you require them. Thank you for looking into this.

Reply:
I'll look into that needle problem to see if it causes real problem.

I have removed one `wait_still_screen` and added comment for the other, can this PR be merged then?

Reply:
Sorry but I will not merge the PR as is. A `wait_still_screen` just before `assert_screen` is wasteful and error-prone because the still time is a gamble: Either we wait enough and waste time or we do not wait enough and hit a race condition.  Unfortunately I do not even understand the comment enough so I don't know what "menu" you are referring to that should disappear. Again, as stated: If you feel existing needles can cause false-matches, no problem to remove them.

Reply:
The menu I'm referring to is `oowriter-menus-file`, it will disappear because of boo#1156745.

This PR is opened to workaround this bug.  So `wait_still_screen` is the core of this workaround.  It will not hit a race condition because when the bug happens, the menu will disappear in less that 1 second.  Without this `wait_still_screen` we will have a race condition: on my fast local machine it will hit a false positive and fail later while on o.o.o it fails directly.

If we want to make the waste less time, we can change the `wait_still_screen` to `sleep 2`.  Again, some sort of wait has to be done because this is a **workaround**, and can be removed when the referenced bug is fixed.

Reply:
As you are suggesting replacing `wait_still_screen` with `sleep 2`: Please read the documentation for the testapi commands: http://open.qa/api/testapi/
At least `wait_still_screen(2)` would be better than sleep 2 :) If you insist on keeping the `wait_still_screen`, please add a lower-than-default value at least.

Reply:
Updated as requested, please review again.

Reply:
ok, you changed the value. But as I stated in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/8958#discussion_r349593837 , we do not know yet if that value is high enough. So if you want to go this route then please use proper https://progress.opensuse.org/projects/openqatests/wiki/Wiki#Statistical-investigation and provide multiple test runs, e.g. 20-100, showing that the problem is gone.

Reply:
As requested, 20 runs passed `sleep 2` here: https://openqa.opensuse.org/tests/overview?build=poo60062_investigation2